commit 6279017e807708a07db5edace462713a93625da3
Author: Will Deacon <will@kernel.org>
Date:   Fri Nov 13 11:38:42 2020 +0000

    KVM: arm64: Move BP hardening helpers into spectre.h
    
    The BP hardening helpers are an integral part of the Spectre-v2
    mitigation, so move them into asm/spectre.h and inline the
    arm64_get_bp_hardening_data() function at the same time.
    
    Signed-off-by: Will Deacon <will@kernel.org>
    Signed-off-by: Marc Zyngier <maz@kernel.org>
    Cc: Marc Zyngier <maz@kernel.org>
    Cc: Quentin Perret <qperret@google.com>
    Link: https://lore.kernel.org/r/20201113113847.21619-6-will@kernel.org

diff --git a/arch/arm64/kvm/hyp/hyp-entry.S b/arch/arm64/kvm/hyp/hyp-entry.S
index 0a5b36eb54b3..874eacdabc64 100644
--- a/arch/arm64/kvm/hyp/hyp-entry.S
+++ b/arch/arm64/kvm/hyp/hyp-entry.S
@@ -13,6 +13,7 @@
 #include <asm/kvm_arm.h>
 #include <asm/kvm_asm.h>
 #include <asm/mmu.h>
+#include <asm/spectre.h>
 
 .macro save_caller_saved_regs_vect
 	/* x0 and x1 were saved in the vector entry */